### PR TITLE
Configurable active plugins (per task quality)

### PR DESCRIPTION
**Pull request by icedevml**
List of plugins in `config.ini` that are activated globally or depending on task's quality.

### CODE REVIEW

**Comment by kscieslinski** [comment]
Wouldn't whitelisting approach be better? PR. looks LGTM

**Review by kscieslinski — APPROVED** [review]

File: drakrun/drakrun/main.py [hunk 1]
```diff
@@ -400,13 +400,18 @@ def process(self):
...
-                               "-x", "poolmon",
-                               "-x", "objmon",
-                               "-x", "socketmon",
-                               "-x", "dkommon",
-                               "-x", "envmon",
+                task_quality = self.current_task.headers.get("quality", "high")
+                excl_str = self.config.config['drakrun'].get('excluded_plugins', 'poolmon, objmon, socketmon, dkommon, envmon')
+                excl_q_str = self.config.config['drakrun'].get(f'excluded_plugins_quality_{task_quality}', '')
+                excl_cmdline = []
+
```
Comment (chivay) [inline drakrun/drakrun/main.py:407]: IMO, this should be moved into initialization step and queried like: `self.disabled_plugins[quality]`.

**Comment by chivay** [comment]
Also, this change should be somehow documented, at least in the default configuration

File: drakrun/drakrun/main.py (hunk 1 above)
Comment (icedevml) [reply drakrun/drakrun/main.py]: olk

**Review by chivay — APPROVED** [review]

**Pull request merged**